src/test/test_controller.c

```diff
@@ -1838,6 +2020,10 @@ struct testcase_t controller_tests[] = {
     NULL },
   { "download_status_consensus", test_download_status_consensus, 0, NULL,
     NULL },
+  {"getinfo_helper_dir_from_cache", test_getinfo_helper_dir_from_cache,
```

Comment:
Please rename these tests to clarify that they test the consensus fetching function and not the whole `getinfo_helper_dir()`.